terraform/maas/rackd_vms.tf

```diff
@@ -0,0 +1,84 @@
+# MAAS MDC1 rackd controller vm
+resource "vsphere_virtual_machine" "mdc1_wintest_rackd_1" {
+  provider = vsphere.mdc1
+
+  name                 = "maas-rackd1.wintest.releng.mdc1.mozilla.com"
+  resource_pool_id     = data.vsphere_resource_pool.mdc1_relops_terraform_resource_pool.id
+  datastore_cluster_id = data.vsphere_datastore_cluster.mdc1_releng_datastore_cluster.id
+
+  annotation = "Managed by Terraform"
+  num_cpus   = 2
+  memory     = 4096
+  guest_id   = data.vsphere_virtual_machine.mdc1_template.guest_id
+
+  folder         = "Relops-Terraform"
+  enable_logging = true
+
+  scsi_type = "lsilogic"
+
+  # wintest.relng.mdc1 vlan
+  network_interface {
+    network_id = data.vsphere_network.mdc1_releng_network_vlan_240.id
+  }
+
+  disk {
+    label          = "disk0"
+    size           = 200
```

Comment:
200GB is due to the fact the rackd controllers will be caching disk images

Reply:
is this provisioned or max? can we set it lower and alert if we fill a limit?

Reply:
Forgive my ignorance, but is that disk thin or thick provisioned?

Reply:
^ thx :cknowles for translating my question to the relevant question for a vm. 🙇

Reply:
Yes, by default disk is thin provisioned but I'm all for declaring things explicitly even when they are defaults.  It makes for better clarity and intent.